### ISSUE TITLE
Request for crediting original authors

### ISSUE THREAD

**Issue by Brohann3214** [Yu-Hayate/Yu-hayates_Mouse#1]
Nice job packaging up the [accelerometer mouse](https://forum.makecode.com/t/announcement-makecode-arcade-prehistoric-game-jam/20995/77?u=brohann) as an extension! Is it too much to ask you if you could credit unsignedardiuno and I in the code somewhere?

**Comment by Yu-Hayate** [comment]
of course! sorry for not responding, i didn't have a phone for the 2FA thing Xd

**Comment by Yu-Hayate** [comment]
added into the README.md file!
thanks for bringing this to my attention, Brohann3214.

**Comment by Yu-Hayate** [comment]
i didn't even noticed you already had the same idea as me, I'm very sorry for "remaking" the extension
do i have your approval to keep the project up?

**Comment by Yu-Hayate** [comment]
I read your message wrong the first 5 times Xd 
* i didn't realise you had the same idea to use the accelerometer extension 
* i also didnt know you had that idea 
* i didnt copy the idea off of you or @unsignedardiuno, sorry for the misunderstanding

**Comment by Brohann3214** [comment]
That's totally okay!  Feel free to keep this extension. I just got a little bit suspicious when nobody had thought of it for the first 4 years of makecode and after I came up with it a similar extension was made. (also your forum post was ai generated😑)

**Comment by Yu-Hayate** [comment]
well that's annoying literary anything i wright looks like AI
might as well quit my job and the forum
(witch a sortof did for a month)

**Comment by Yu-Hayate** [comment]
well that's annoying literary anything i wright looks like AI
might as well quit my job 
(which I sort of did for a month)
